src/Markdig/MarkdownPipeline.cs

```diff
@@ -52,7 +52,7 @@ internal MarkdownPipeline(OrderedList<IMarkdownExtension> extensions, BlockParse
         /// True to parse trivia such as whitespace, extra heading characters and unescaped
         /// string values.
         /// </summary>
-        public bool TrackTrivia { get; set; }
+        internal bool TrackTrivia { get; set; }
```

Comment:
Leave it public getter - otherwise once building a pipeline, we can't know if it's asking trivia or not.